index.js

```diff
@@ -67,6 +77,63 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', event => {
+    console.log(event.target)
+    toggleColor(event.target)
+})
+
 /***** Deliverable 2 *****/
 
+likeButton = document.querySelector("button.like-button")
+likeCount = document.querySelector("p.likes")
+
+count = 1000
+likeButton.addEventListener('click', event => {
+    // My appraoch
+    // likeButton.classList.toggle('liked')
+    // count += 1;
+    // likeCount.innerHTML = "Likes: " + count
+    // APPROACH 1
+    const likesPDisplay = document.querySelector('p.likes')
+    traveler.likes += 1
+    likesPDisplay.textCOntent = `${traveler.likes} Likes`
+    console.log('clicked', event.target)
+
+    APPROACH 2
+    const likesPDisplay = document.querySelector('p.likes')
+    const currLikes = parseInt(likesPDisplay.textContent)
+    likesPDisplay.textContent = `${currLikes + 1} Likes`
+})
+
+// Approach 3
+console.log(event.target)
+const likesPDisplay = event.target.previousElementSibling
```

Comment:
and this!! let's dig into it more how we could incorporate both:

1. find the `p` tag where the current number of likes is stored on the page
2. extract the number out of the string, make sure to convert it to a number with `parseInt()`
3. add 1 to that number.
4. Create a new string with the new number
5. update the `p` tag to have the newly created string as its `innerText`.
```javascript
const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling;
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```
- Or with ES6 notation
```javascript
const likeBtn = document.querySelector(".like-button");
const addNewLike = e => {
    const likesPTag = e.target.previousElementSibling;
    let currentLikes = parseInt(likesPTag.innerText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```